browser/help/topics/pext.md

```diff
@@ -30,7 +30,7 @@ CHROM    POS   REF  ALT   CONSEQUENCES
 Therefore it's important to note that **the base-level pext value represents a maximum of the position, and will always be higher than the annotation-level pext value**. Therefore just because a position has a high base-level pext value doesn't mean a pLoF annotated at that position will have a high annotation-level pext value.
 
 ### Caveats
-We note that the pext values are derived from isoform quantifications using the RSEM tool. Isoform quantification tools can be imprecise, especially for longer genes with many annotated isoforms. While we have manually curated a set of regions with low pext values, and find that they are enriched for annotation errors, domain knowledge of a gene will outperform this summary metric (ie. there may be edge cases for which an exon that is established to be critical for gene function may appear unexpressed with pext). We also note that the GTEx dataset is postmortem adult tissue, and thus we cannot dismiss the probability that an exon may be development-specific or may be expressed in tissues not represented in GTEx.
+We note that the pext values are derived from isoform quantifications using the RSEM tool. Isoform quantification tools can be imprecise, especially for longer genes with many annotated isoforms. While we have manually curated a set of regions with low pext values, and find that they are enriched for annotation errors, domain knowledge of a gene will outperform this summary metric (i.e., there may be edge cases for which an exon that is established to be critical for gene function may appear unexpressed with pext). We also note that the GTEx dataset is postmortem adult tissue, and thus we cannot dismiss the probability that an exon may be development-specific or may be expressed in tissues not represented in GTEx. In addition, tissues with fewer than 50 samples in GTEx v10 were not included in gnomAD v4 pext calculations.
```

Comment:
Looks good! 

Not sure if we should include [this](https://github.com/broadinstitute/gnomad_qc/blob/ac358b4920d7f8e0ec49cde771bffde84536582e/gnomad_qc/analyses/compute_pext.py#L431) info too?

```
    parser.add_argument(
        "--browser-min-samples",
        help=(
            "Minimum number of samples to include a tissue in the individual tissue "
            "pext values displayed as a track on the browser."
        ),
        type=int,
        default=100,
    )
```

Reply:
good idea, will add!